jekyll/_cci2/code-coverage.md

```diff
@@ -135,7 +220,80 @@ coverage html  # open htmlcov/index.html in a browser
 The generated files will be found under `htmlcov/`, which can be uploaded in a
 `store_artifacts` step in your config:
 
+{:.tab.python_example.Cloud}
+```yaml
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+    - image: cimg/python:3.10-browsers
+      auth:
+        username: mydockerhub-user
+        password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+    - checkout
+    - run:
+        name: Setup testing environment
+        command: |
+          pip install '.[test]' --user
+          echo $HOME
+    - run:
+        name: Run Tests
+        command: |
+          $HOME/.local/bin/coverage run -m pytest
+          $HOME/.local/bin/coverage report
+          $HOME/.local/bin/coverage html  # open htmlcov/index.html in a browser
+    - store_artifacts:
+        path: htmlcov
+workflows:
+  version: 2
+  workflow:
+    jobs:
+    - build
+```
+
+{:.tab.python_example.Server_3}
+```yaml
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+    - image: cimg/python:3.10-browsers
+      auth:
+        username: mydockerhub-user
+        password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+    - checkout
+    - run:
```

Comment:
If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`